dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderTelemetry.h

```diff
@@ -18,57 +18,62 @@ class AppNotificationBuilderTelemetry : public wil::TraceLoggingProvider
 public:
 
     BEGIN_COMPLIANT_MEASURES_ACTIVITY_CLASS(BuildNotification, PDT_ProductAndServicePerformance);
-    DEFINE_ACTIVITY_START(NotificationTelemetryHelper& notificationTelemetryHelper)
-    {
-        TraceLoggingClassWriteStart(
-            BuildNotification,
-            _GENERIC_PARTB_FIELDS_ENABLED,
-            TraceLoggingBool(notificationTelemetryHelper.IsPackagedApp(), "IsAppPackaged"),
-            TraceLoggingWideString(notificationTelemetryHelper.GetAppName().c_str(), "AppName"));
-    }
+        DEFINE_ACTIVITY_START(NotificationTelemetryHelper& notificationTelemetryHelper) noexcept try
```

Comment:
Did we use try CATCH_LOG pattern in other helpers too?

Reply:
Do you mean if we use try and CATCH_LOG on all providers? If so, then yes, we do.